custodian/lobster/jobs.py

```diff
@@ -78,16 +78,16 @@ def setup(self, directory="./"):
 
     def run(self, directory="./"):
         """Runs the job."""
-        cmd = self.lobster_cmd
+        cmd = " ".join(self.lobster_cmd)  # join split commands (e.g., from atomate and atomate2)

```

Comment:
do we want to support both `str` and `Sequence` for `lobster_cmd`?

Reply:
Yes, I think it would be good. Just made a comment on that with two alternatives. You were faster :D